src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -7,6 +7,12 @@ const AdParamKeys = {
   MARKER: 'marker',
 };
 
+const KeyCodes = {
```

Comment:
rename to arrow keycodes or something similar. it isn't very apparent what right and left are

Reply:
changed